hack/test-integration.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 set -e
 
-make PREFIX=/usr install
-
-echo "cd ./integration;" go test $TESTFLAGS ${BUILDTAGS:+-tags "$BUILDTAGS"}
+echo "cd ./integration;" go test "$TESTFLAGS" ${BUILDTAGS:+-tags "$BUILDTAGS"}
 cd ./integration
-go test $TESTFLAGS ${BUILDTAGS:+-tags "$BUILDTAGS"}
\ No newline at end of file
+go test "$TESTFLAGS" ${BUILDTAGS:+-tags "$BUILDTAGS"}
```

Comment:
The `"$TESTFLAGS"` changes are inconsistent with how `Makefile` uses the variable, that explicitly documents using a value with spaces to mean separate arguments.

Reply:
ack, i'll check and update. Thanks